### PR TITLE
React UI: batch of fixes

### LINKED ISSUE #1416: CVAT new UI: add a hint how to delete a point on a polygon
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour

Need to have the information that we can delete a point on a polygon using ctrl+double mouse click
### Current Behaviour

No information that there is an opportunity to delete a point on a polygon using ctrl+double mouse click. The context menu of a point contains the option "delete point"
### Possible Solution

Add a hint description in the context menu near the "delete point" (ctrl+2lmb click)

### LINKED ISSUE #1419: CVAT new UI: open some buttons in a new tab or window
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour

The following buttons and links should have and option to open them in a new tab or window:
1. CVAT dashboard, the "open task" button:
image
2. The list of jobs inside a task:
image
3. Inside a CVAT job. "Menu"- "Open the task" button:
image

### Current Behaviour

All the described buttons and links can't be open in a new tab or window
### Possible Solution

Add an option to open them in a new tab or window using the mouse wheel or right mouse button

### Context

Very often need to open several CVAT tasks and jobs simultaneously to find some information

### LINKED ISSUE #1420: CVAT new UI: Attribute Annotation Mode. To hide the right panel with attributes
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour

A button to hide this panel like in the standard mode:

### Current Behaviour

Shown above in the gif
### Possible Solution

Shown above in the gif

### Context

It will be useful to hide this panel to get better concentration on attributes. In most cases there are 2-4 attributes and it's simple to remember their meanings.

### LINKED ISSUE #1434: CVAT new UI: make the "lock" icon more informative
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour

Easy-recognizable icon for the "lock" property
### Current Behaviour

Visually not clear that an object is locked. The icon is too small and uninformative:
Not locked:
image
Locked:
image

### Possible Solution

Enlarge this icon or predict visual changes for conditions (full fill)

### LINKED ISSUE #1500: CVAT new UI: display of cuboids
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour

The look of Cuboid makes editing clear and convenient.

### Current Behaviour

If “occluded” is turned on, cuboid points have a dashed line, and projection lines also become dashed

image5529-5

Cuboid planes do not change the fill on hover (unlike the old interface)

### Possible Solution

When turned on, the “occluded” projection lines and the stroke of the points do not become dashed.

Cuboid planes on hover become highlighted (like in the old UI)

### Steps to Reproduce (for bugs)

N/A

### Context

N/A
### Your Environment

- Git hash commit (`git log -1`):
- Docker version `docker version` (e.g. Docker 17.0.05):
- Are you using Docker Swarm or Kubernetes?
- Operating System and version (e.g. Linux, Windows, MacOS):
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

### PR DESCRIPTION
**Pull request by ActiveChooN**
### Motivation and context

This PR fixes several issues:
Resolve #1419 
Resolve #1500 
Resolve #1420 
Resolve #1434 
Resolve #1416 
Fixed cuboid reset perspective does not emit `canvas.edited` event

### How has this been tested?

Manually

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by nmanovic** [comment]
@ActiveChooN , could you please resolve conflicts?

File: cvat-canvas/src/typescript/svg.patch.ts [hunk 1]
```diff
@@ -263,10 +263,10 @@ function getTopDown(edgeIndex: EdgeIndex): number[] {
             this.rbProj = this.line(this.updateProjectionLine(this.cuboidModel.rb.getEquation(),
                 this.cuboidModel.rb.points[1], this.cuboidModel.vpr));
 
-            this.ftProj.stroke({ color: '#C0C0C0' });
-            this.fbProj.stroke({ color: '#C0C0C0' });
-            this.rtProj.stroke({ color: '#C0C0C0' });
-            this.rbProj.stroke({ color: '#C0C0C0' });
+            this.ftProj.stroke({ color: '#C0C0C0' }).addClass('cvat_canvas_shape_projections');
```
Comment (bsekachev) [inline cvat-canvas/src/typescript/svg.patch.ts:266]: Let's rename to ``cvat_canvas_cuboid_projections``

File: cvat-canvas/src/typescript/canvasView.ts [hunk 2]
```diff
@@ -437,10 +440,22 @@ export class CanvasViewImpl implements CanvasView, Listener {
...
+                        const points = translate(pointsToArray((e.target as any)
+                            .parentElement.parentElement.instance.attr('points')));
+                        self.onEditDone(
+                            state,
+                            points,
+                        )
+                        e.preventDefault();
+                        return;
+                    }
+                }
```
Comment (bsekachev) [inline cvat-canvas/src/typescript/canvasView.ts:458]: Is this to reset perspective?

File: cvat-ui/src/components/annotation-page/attribute-annotation-workspace/attribute-annotation-sidebar/attribute-annotation-sidebar.tsx [hunk 3]
```diff
@@ -218,6 +222,16 @@ function AttributeAnnotationSidebar(props: StateToProps & DispatchToProps): JSX.
...
             <Layout.Sider {...siderProps}>
+                {/* eslint-disable-next-line */}
+                <span
+                    className={`cvat-objects-sidebar-sider
+                        ant-layout-sider-zero-width-trigger
+                        ant-layout-sider-zero-width-trigger-left`}
+                    onClick={() => setSidebarCollapsed(!sidebarCollapsed)}
+                >
+                    {sidebarCollapsed ? <Icon type='menu-fold' title='Show' />
+                        : <Icon type='menu-unfold' title='Hide' />}
```
Comment (bsekachev) [inline cvat-ui/src/components/annotation-page/attribute-annotation-workspace/attribute-annotation-sidebar/attribute-annotation-sidebar.tsx:233]: It occludes filters field. Doesn't look pretty

File: cvat-ui/src/components/annotation-page/top-bar/annotation-menu.tsx [hunk 4]
```diff
@@ -20,6 +20,7 @@ interface Props {
     dumpActivities: string[] | null;
     exportActivities: string[] | null;
     installedReID: boolean;
+    taskID: number
```
Comment (bsekachev) [inline cvat-ui/src/components/annotation-page/top-bar/annotation-menu.tsx:23]: Missed semicolon

**Comment by bsekachev** [comment]
I still unable to open task/job in open tab using ctrl+click. At least on ubuntu in both chrome and firefox. Are you able to?

File: cvat-canvas/src/typescript/canvasView.ts (hunk 2 above)
Comment (ActiveChooN) [reply cvat-canvas/src/typescript/canvasView.ts:458]: @bsekachev, yes

Comment (bsekachev) [reply cvat-canvas/src/typescript/canvasView.ts:458]: @ActiveChooN 

Two different event listener are responsible for the same feature. It makes me confused

**Comment by ActiveChooN** [comment]
@bsekachev, and you cannot. But there is ability to open link in new tab with context menu. I guess it's good trade-off between router links and ordinary links. Are there any other good solutions?

**Comment by bsekachev** [comment]
@ActiveChooN 

Okay, at least I can open in new tab with mousewheel.

File: cvat-canvas/src/typescript/svg.patch.ts [hunk 5]
```diff
@@ -442,6 +450,7 @@ function getTopDown(edgeIndex: EdgeIndex): number[] {
                 accumulatedOffset.x = 0;
                 accumulatedOffset.y = 0;
                 const resizedFacePoint = getResizedPointIndex(event);
+                // @ts-ignore: ts(2339)
```
Comment (bsekachev) [inline cvat-canvas/src/typescript/svg.patch.ts:453]: What is wrong here?

File: cvat-ui/package-lock.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cvat-ui",
-  "version": "1.0.1",
+  "version": "1.1.2",
```
Comment (bsekachev) [inline cvat-ui/package-lock.json:3]: Hmm.. Why 1.1.2? Right now in develop I see 1.0.1
Probably it should be version 1.0.2

**Pull request merged**